### ISSUE TITLE
Support deep linking to the debug tab on a given isolate and library

### ISSUE THREAD

**Issue by natebosch** [flutter/devtools#4241]
In the test runner we currently link to the Observatory UI when debugging tests. We'd like to point to the devtools UI instead.
https://github.com/dart-lang/test/issues/1563

Observatory lets us link to the `/inspect` path with an `isolateId` query parameter, and a `objectId` parameter pointing the `_test.dart` library.

**Comment by kenzieschmoll** [comment]
At the point that we want to put together a deep link, do we have the vm service uri for the running test?
If so, we'd just need to add query param support for 'isolateId' and 'objectId' (or maybe 'scriptId' if we want to open a specific script in the debugger).
@elliette

**Comment by natebosch** [comment]
> At the point that we want to put together a deep link, do we have the vm service uri for the running test?

Yes. We have the VM service URI, as well as the ID of the isolate and the library.

http://google3/dart/build_defs/dart_codegen_rule/rules.bzl;l=8-18;rcl=459397204
